e2e/BUILD.bazel

```diff
@@ -17,6 +17,7 @@ protractor_web_test_suite(
     name = "prodserver_test",
     data = [
         "@angular//packages/bazel/src/protractor/utils",
+        "@npm//@angular/bazel",
```

Comment:
maybe a comment that this dependency gives us the protractor utils?

Reply:
👍

Reply:
And forgot to remove `"@angular//packages/bazel/src/protractor/utils",` as well :)